dmutils/dates.py

```diff
@@ -48,3 +50,18 @@ def _get_todays_date():
 
 def _get_length_of_brief(brief):
     return brief['requirementsLength'] if brief.get('requirementsLength') else '2 weeks'
+
+
+def update_framework_with_formatted_dates(framework):
```

Comment:
Is this perhaps a bit prescriptive? What if for some reason we want to display the same date differently in two places? As we have the Jinja filters, can we not just handle the conversion on the frontend?

Reply:
Although I guess you still have the unformatted date in the record. Why do we need this method then?

Reply:
It is a little prescriptive, you're right. But it will hopefully keep these things consistent. When we were using the `dates.yml` file, we knew that everywhere we displayed that date it would be in a single format. This helps us achieve the same consistency, but we also have the flexibility to use the raw UTC datetime to get to any other format if we _need_ to.

Having this here helps us be consistent unless we have a good reason not to be, I think.

Reply:
If you don't agree with that logic though, we could talk about taking it out. I'm not totally attached to this, but it made sense to me insofar as trying to preserve the existing format as much as possible.

Reply:
I guess if we know we're going to be using this particular set of dates in this particular format a bunch, then having this makes sense. If it's only going to be used in one place I think it could be a bit unnecessary. However we've got it now, so lets keep it!